Dockerfile

```diff
@@ -1,20 +1,25 @@
-FROM cern/cc7-base:20190724
-COPY ./requirements.txt /cms-htcondor-es/requirements.txt
-RUN yum install -y git sudo python36 python36-virtualenv python36-pip && \
-ln -fs /usr/bin/python3 /usr/bin/python && \
-ln -fs /usr/bin/pip3.6 /usr/bin/pip &&\
-pip install -r /cms-htcondor-es/requirements.txt
+FROM cern/cc7-base:20210501-2.x86_64
+
+# Build python3.8
+RUN yum install -y git gcc openssl-devel bzip2-devel libffi-devel make wget
```

Comment:
make all RUN steps below as a single step, e.g.
```
RUN yum install -y git gcc openssl-devel bzip2-devel libffi-devel make wget && \
         cd /opt && wget https://www.python.org/ftp/python/3.8.7/Python-3.8.7.tgz && tar xzf Python-3.8.7.tgz && \
         cd /opt/Python-3.8.7 && ./configure --enable-optimizations && make altinstall && \
         ln -fs /usr/local/bin/python3.8 /usr/bin/python && ln -fs /usr/local/bin/pip3.8 /usr/bin/pip && rm -rf /opt/Python-3.8.7

```
This will reduce by 3 steps time spend in docker layers.